molecule/default/tests/test_default.py

```diff
@@ -1,25 +1,76 @@
 import os
 import re
-
 import pytest
 import testinfra.utils.ansible_runner
 
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('cinder_volume')
 
 
-@pytest.mark.skip(reason='TODO: will update the test to run '
-                         'cinder command on controller')
+# TODO: will update the test to run cinder command on controller node
+# TODO: since the test cannot run on storage hosts.
+@pytest.mark.skip(reason='this test can not run on storage host')
 def test_cinder_service(host):
     cmd = "sudo bash -c \"source /root/openrc; cinder service-list\""
     output = host.run(cmd)
     assert ("cinder-volume" in output.stdout)
 
 
+##########################################################################
+# ASC-222: Verify the Filesystems were correctly provisioned for Cinder  #
+##########################################################################
+
+# Test 2a: Check the Cinder Nodes
```

Comment:
The Pythonic way of documenting is to use [docstrings](http://www.pythonforbeginners.com/basics/python-docstrings) instead of comments above functions/classes.